integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -4849,7 +4849,7 @@ mod lower_storage_key_limit_test {
             deploy_test_contract(
                 &mut env,
                 "test0".parse().unwrap(),
-                near_test_contracts::rs_contract(),
+                near_test_contracts::base_rs_contract(),
```

Comment:
please add the comment mentioning that it is using base_rs_contract on purpose

Reply:
I think that it should be commented in near-test-contracts readme, because `base_rs_contract` is used in other tests as well.
I will merge it now and add comment in separate PR, because pipeline is failing now for all other PRs

Reply:
Hm, I am not sure what's the best way to "teach" the API here. There are `rs_contract()`, `base_rs_contract()` and `nightly_rs_contract()` with different semantics, which are documented in their respective docstrings. I don't think we should comment every call-site with "we are using `base_rs_contract` here intentionally" -- the fact that you are using `something_rs_contract` rather `rs_contract` already signals deliberate intent.

Let me maybe add specific guidance about upgradability tests to the docs as a use-case where base_ matters.

Reply:
https://github.com/near/nearcore/pull/6642